nbi/engine/native/launcher/windows/src/JavaUtils.c

```diff
@@ -138,7 +139,7 @@ JavaVersion * getJavaVersionFromString(char * string, DWORD * result) {
             major = (major) * 10 + c - '0';
             if (major > 999) return vers;
             continue;
-        } else if(c=='.'){
+        } else if(c=='.' || c=='+'){
```

Comment:
Why is this ?

Reply:
Without this, method returns null for JDK 12 version string 12+33
$ java -version
java version "12" 2019-03-19
Java(TM) SE Runtime Environment (build 12+33)
Java HotSpot(TM) 64-Bit Server VM (build 12+33, mixed mode, sharing)

Reply:
This doesn't set the build number in version, I'll handle this.